src/lib/RbtPharmaSF.cxx

```diff
@@ -189,13 +189,13 @@ void RbtPharmaSF::ScoreMap(RbtStringVariantMap& scoreMap) const {
         // Store the mandatory constraint scores
         for (RbtUInt i = 0; i < m_conScores.size(); i++) {
             ostringstream field;
-            field << name << ".con_" << i + 1 << ends;
+            field << name << ".con_" << i + 1 << ">" << ends;
```

Comment:
the `>` character should be added by the mdl file sink class, but the `ends` is interfering. I think the fix is to just remove the `ends`.
Please give this a try
```suggestion
            field << name << ".con_" << i + 1;
```